### ISSUE TITLE
"Report user" functionality would be useful.

### ISSUE THREAD

**Issue by thouis** [mtgred/netrunner#4559]
There doesn't seem to be a way to report users for bad behavior.  I went looking for this functionality after seeing the NorwegianLegion username debacle in chat (see below).

![image](https://user-images.githubusercontent.com/473043/66703963-e0852980-ece5-11e9-84d7-3ac0f7263cb8.png)

**Comment by NoahTheDuke** [comment]
Sadly, we don’t have any mod tools at the moment and no mods. The only thing I can do is delete those messages, but I’m not online currently. It’s been on the to do list for a while.

**Comment by thouis** [comment]
Having been involved with moderating a game site a few years back, I can only wish you luck. 

If I get a chance, would it be worth me looking at adding a Report button?  Would that be the best route to go?  Or would it just add to the difficulties of running the site?

Also, the username itself is a problem, not just the messages.

**Comment by NoahTheDuke** [comment]
Yeah, I just looked that up. That's real concerning. I'll see what I can do.
